Code/sym.py

```diff
@@ -0,0 +1,34 @@
+# SE HW - 2
+
+class Sym:
+    def __init__(self, column_name, column_position):
+        self.column_name = column_name
+        self.column_position = column_position
+        self.data = list()
```

Comment:
Here, I assume your data, is 'n' in lua code. n is a number, not list. It represents the count of the total no. of items encountered so far.

Reply:
As discussed, this will be taken up on a different thread.